pageserver/src/tenant/timeline/eviction_task.rs

```diff
@@ -296,6 +304,7 @@ impl Timeline {
                 if stats.candidates == stats.not_evictable {
                     debug!(stats=?stats, "eviction iteration complete");
                 } else if stats.errors > 0 || stats.not_evictable > 0 {
+                    // do not warn on timeouts, in overloaded situations they are normal
                     warn!(stats=?stats, "eviction iteration complete");
```

Comment:
It's an abnormal condition, we should warn in the case of eviction timeouts.
We want it to stand out in the logs.
The level warn vs info doesn't make a difference to log volume.

Reply:
Oki, made it a warning.